config/tranformers/name-path-to-dot-notation.ts

```diff
@@ -0,0 +1,25 @@
+import StyleDictionary from 'style-dictionary'
+
+/**
+ * capitalize
+ * @description capitalizes bur keeps the rest of the word as is
+ * @todo: replace with _.capitalize once we don't use camelCase in token names anymore
+ * @param param0 
+ * @returns 
+ */
+const capitalize = ([firstLetter, ...restOfWord]: string): string => {
+  return firstLetter.toUpperCase() + restOfWord.join("")
+}
+
+
+const camelCase = (string: string): string => {
```

Comment:
Should capitalize and camelCase be in this file? or hoisted out?

Reply:
Well in this case they are special:

- capitalize does not, e.g. like the lodash version, convert everything to lowercase. It only changes the first letter of the word, which may be unexpected but is need for keeping out names like `onEmphasis` in tact.

- camelCase is very specific in that it only combines the parts separated by space, `-` or `_`

Because of this I think keeping it internal would be good. 

But once we have the naming convention figured out and `eslint` for json set up and can be sure we get only specific things we could look into replacing it with different functions.

Maybe we should rename `capitalize` to `ucfirst`?

Reply:
Makes sense 👍 

In that case, can we add these as comments / annotations above the functions please? So it's clear to others these are non-standard?

Also agree on changing naming.. I'd recommend going verbose and not abbreviating anything. E.g. `lowerCaseFirstCharacter`.

Reply:
Done.